student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
```

Comment:
### [OPTIONAL]
Use require when you want to use external gems. Use require_relative for local files relative to the current working directory.